close_test.go

```diff
@@ -433,14 +432,16 @@ func TestCloseSendError(t *testing.T) {
 	clientCh := testutils.NewClient(t, nil)
 
 	// Create a connection that will be shared.
-	require.NoError(t, testutils.Ping(clientCh, serverCh), "Ping from client to server failed")
+	// Use PingLong here so that we have a context with a bigger timeout
+	require.NoError(t, testutils.PingLong(clientCh, serverCh), "Ping from client to server failed")
```

Comment:
you don't care about a long timeout here, since it's a call that returns immediately anyway. This should be reverted back to the `testutils.Ping`